src/test/resources/feature/SignInScenarios.feature

```diff
@@ -0,0 +1,32 @@
+@Anatolie
+Feature: Sign in and Forgot password
+
+  Scenario: Login validation with correct data
+    Given User is on "Authentication" page
+    When User introduce "test01@test.com" into "Email address" field
+    And User introduce "1@aBc" into "Password" field
+    And User clicks on "Sing in" button
+    Then User is redirected to "My account" page
+
+
+  Scenario Outline: Verify error handling (Negative scenario)
+    Given User is on "Authentication" page
+    When User introduce "<email>" into "Email address"
+    And User introduce "<password>" into "Password"
+    And User clicks on "Sing in" button
+    Then The error message is displayed
+
+    Examples:
+      | email         | password |
+      | test01@       |          |
+      |               |          |
+      | @             | 1        |
+      | test@test.com |          |
+
+
+  Scenario: Verify Retrieve Password button
+    Given User is on "Authentication" page
+    And User is redirected to "Forgot your password" page by clicking "forgot your password" link
```

Comment:
Could you please separate this step for two in order to handle clicking and redirection functionality separately. For example:
And user clicks "forgot your password" link
And user is redirected to "Forgot your password" page